ksql-common/pom.xml

```diff
@@ -56,6 +56,12 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
Does this mean that every new module will have to explicitly exclude `hamcrest-core`?

Reply:
Yes. In favour of hamcrest-all